adam/language_specific/english/english_language_generator.py

```diff
@@ -128,9 +149,14 @@ def _translate_object_to_noun(
             lexicon_entry = self._unique_lexicon_entry(
                 _object.ontology_node  # pylint:disable=protected-access
             )
-            dependency_node = DependencyTreeToken(
-                lexicon_entry.base_form, lexicon_entry.part_of_speech
-            )
+            if count > 1:
```

Comment:
Note that a lot of the code on the two branches is shared.  Better would be:

```python
if count > 1:
   word_form = lexicon_entry.plural_form
else:
   word_form = lexicon_entry.base_form

dependency_node = DependencyTreeToken(
                    word_form, lexicon_entry.part_of_speech
                )
```

Reply:
Done